src/Components/Card/TeamMemberCard/TeamMember.js

```diff
@@ -1,8 +1,8 @@
-import empty from '../../../assets/images/empty.png';
-import bassam from '../../../assets/images/bassam.png';
-import jana from '../../../assets/images/jana.png';
-import osama from '../../../assets/images/osama.png';
-import ibrahim from '../../../assets/images/ibrahim.png';
+import empty from '../../../Assets/images/empty.png';
```

Comment:
change `Assets` to `assets` everywhere

Reply:
if that's causing an error for you change your `Assets` folder name to lower case